kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/Food2Door/OrderRequest.java

```diff
@@ -0,0 +1,29 @@
+package com.kodilla.good.patterns.challenges.Food2Door;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.HashMap;
+
+public class OrderRequest {
+    private final Producer producer;
+    private final HashMap<Products, BigDecimal> productsOrdered;
```

Comment:
Czemu tutaj użyłeś typu `BigDecimal`? To ma być ilość sztuk każdego produktu w zamówieniu?